addOns/scripts/src/main/resources/org/zaproxy/zap/extension/scripts/resources/Messages.properties

```diff
@@ -105,6 +106,8 @@ scripts.script.load.charset.selected.error = Unable to properly read the script
 
 scripts.type.extender					= Extender
 scripts.type.extender.desc				= Extender scripts add new functionality, including graphical elements and new API end points.\n\n\
+To install a script from the library, right click it and choose \n\
```

Comment:
- I don't think we should add this here. Why just this type and not all of the others? IMO it should be mentioned in all of them (e.g. inject this message before the type description in all).
 - `library` → `templates`

Reply:
That's a good idea!